torch/testing/_internal/distributed/rpc/jit/rpc_test.py

```diff
@@ -41,10 +52,352 @@ def forward(self):
         return self.a
 
 
+# Define Script functions on both client and server sides.
+@torch.jit.script
+def two_args_two_kwargs(
+    first_arg,
+    second_arg,
+    first_kwarg=torch.tensor([3, 3]),
+    second_kwarg=torch.tensor([4, 4]),
+):
+    return first_arg + second_arg + first_kwarg + second_kwarg
+
+
+@torch.jit.script
+def assorted_types_args_kwargs(
```

Comment:
Since we are using RRef in rpc calls a lot, let's include a RRef param here as well

Reply:
@zzzwen since I am using py3 type annotation here. We need to wait for RRef to be a type annotation, right?